app/src/modules/analytics/events/misc/moreInfo.ts

```diff
@@ -0,0 +1,12 @@
+import { MORE_INFO } from "./constants";
+import { trackEvent } from "modules/analytics";
+
+export const trackMoreInfoViewed = (rule_type: string, source: string) => {
```

Comment:
MoreInfo should not be tied only to rules. 
We can have single param - `context`. Eg: `Redirect_source_filters`.